packages/types/src/bellatrix/sszTypes.ts

```diff
@@ -222,7 +222,7 @@ export const SSEPayloadAttributesCommon = new ContainerType(
   {
     proposerIndex: UintNum64,
     proposalSlot: Slot,
-    proposalBlockNumber: UintNum64,
+    parentBlockNumber: UintNum64,
```

Comment:
I am pretty sure more upstream changes are required here as well, passed on previous name I would expect we are setting the block number instead of the parent block number.

Reply:
Good point.
The only use of it is in [getPayloadAttributesForSSE](https://github.com/ChainSafe/lodestar/blob/b1848ec145421a47df8efaff8c16872e391fc1fb/packages/beacon-node/src/chain/produceBlock/produceBlockBody.ts#L522). It's value is currently set to `prepareState.latestExecutionPayloadHeader.blockNumber + 1`. `parentBlockHash` value is calculated in [getExecutionPayloadParentHash](https://github.com/ChainSafe/lodestar/blob/b1848ec145421a47df8efaff8c16872e391fc1fb/packages/beacon-node/src/chain/produceBlock/produceBlockBody.ts#L461) as `state.latestExecutionPayloadHeader.blockHash` when post-merge. 

How should `parentBlockNumber` value be changed then? And do we have to consider pre-merge? (current value of `proposalBlockNumber` doesn't)

Maybe @g11tech who did the original [work](https://github.com/ChainSafe/lodestar/pull/5281) can advise on that?

Reply:
> And do we have to consider pre-merge?

Pre-merge the consensus layer does not have access to this data and `SSEPayloadAttributes`ssz type was introduced in bellatrix

Reply:
Pulled the change out of this PR
- https://github.com/ChainSafe/lodestar/pull/6471